src/libs/nav.test.ts

```diff
@@ -17,3 +36,58 @@ describe('isTerraNavKey', () => {
     expect(result).toBe(false);
   });
 });
+
+describe('goToPath', () => {
+  it('does not include query params if not provided', () => {
+    const pathname = '/url-version-of-path/foo/bar';
```

Comment:
Nit: for consistency with other tests, please comment the "Arrange" section as well as the "Act" and "Assert" sections. Same comment for other tests here.

```suggestion
    // Arrange
    const pathname = '/url-version-of-path/foo/bar';
```